### ISSUE TITLE
I got an error after I installed your app from the ownCloud market

### ISSUE THREAD

**Issue by Snoobz** [owncloud/security#36]
Hello, this issue is apparently appeared after I installed you app from the ownCloud market. I got the yellow banner telling me that something is wrong with my configuration of ownCloud. Here is the message:

```
Technical information
=====================
The following list covers which files have failed the integrity check. Please read
the previous linked documentation to learn more about the errors and how to fix
them.

Results
=======
- security
	- INVALID_HASH
		- templates/settings-personal.php

Raw output
==========
Array
(
    [security] => Array
        (
            [INVALID_HASH] => Array
                (
                    [templates/settings-personal.php] => Array
                        (
                            [expected] => 9d6bad0bd3fa7e9a8fdd4fa6d3670754b898996eb980a19299a3a72f345a52b5570701c485d1872e668f594f229a4a578abf8650223be64afc8dbbf160692acf
                            [current] => 39536879cf9dcd358fb409413dfc46ce3dc09e82c3ed67abb528a005088d2bef1adeef32edd1cfb5cc0ad0bcb75befe35a25458869d92c74e53e5e3c7878e716
                        )

                )

        )

)
```
I need some help, when I check the folder /template (/my/path/to/owncloud/core/template
there is no file named settings-personal.php
I got there a lot of .php file but none named settings-personnal.php I guess you need to create one that then I'll need to put it there to fix that issue. If someone could provide me the code of that .php file or update the market app I'll appreaciate it!

And thank you for your work, great app that you made there.

**Comment by karakayasemi** [comment]
Hi @Snoobz , the mentioned file in here: https://github.com/owncloud/security/blob/master/templates/settings-personal.php .

I will try to reproduce your error in my instance when I find a time, maybe something went wrong in packaging.  If your file different than repo, you can simply copy-paste the mentioned file from repo to your server, for now.

**Comment by Snoobz** [comment]
Hello @karakayasemi the problem is now solved! No need to reproduce this issue because there is none. Let me explain, I modified this file (settings-personal.php, available at /my/path/to/owncloud/apps/security/templates) and a few days ago I changed a few strings in this file to have the sentences in french (if you know what I mean) and apparently ownCloud didn't like that words between the " " were in french. So I copy/paste your code and everthing works fine now.

PS: In the original post I made a mistake saying a file was missing in /my/path/to/owncloud/core/template There is no need to add a settings-personal.php file there, my mistake.

Thank you!
